### DISCUSSION TITLE
Failed to Resolve IOR

### DISCUSSION THREAD

**Discussion by kubraturhan** [OpenDDS/OpenDDS#4473] [Q&A]
While I am creating domain participant with DomainParticipantFactoryOperations create_participant method, I get the error "failed to resolve ior-". I share the ini file and the terminal output. Can you help me please? Is there any error in ini file or anything else?

![error2](https://github.com/OpenDDS/OpenDDS/assets/3930719/e022ed1d-74de-469b-812d-bf0b5a9f68c5)

![error](https://github.com/OpenDDS/OpenDDS/assets/3930719/d1cbf3da-5a3c-4b88-8fee-b09722ff3bea)

**Comment by jrw972** [comment]
Is the InfoRepo running?

**Comment by kubraturhan** [comment]
Thank you four your reply. InfoRepo was not working beacuse of VCRedist was not installed in my PC. After installation InfoRepo started working and the error disappeared. But now I have another error and I could not resolve it. After I successfully created a topic via create_topic method of DomainParticipantTaoPeer, I call create_datawriter method of PublisherTaoPeer and the java application is crashing. I share the code and the error logs. Can you help me please?
![code](https://github.com/OpenDDS/OpenDDS/assets/3930719/a1309fb6-db0f-496d-9c16-d4a0e14f3537)
![Exception](https://github.com/OpenDDS/OpenDDS/assets/3930719/36aeee0c-de68-4edf-b2ce-c0887571d04b)
![StackTrace](https://github.com/OpenDDS/OpenDDS/assets/3930719/0801b278-88bc-4088-84ff-d1ee671548ef)

**Comment by mitza-oci** [comment]
> But now I have another error and I could not resolve it. 

Please start a new discussion for this.
